{
    "dataservicehandler.delete.status.authentication_failed": {
        "message": "Authentication failed, check API key."
    }, 
    "dataservicehandler.delete.status.launch_fail": {
        "message": "Failed to launch the remote API."
    }, 
    "dataservicehandler.delete.status.no_auth_key": {
        "message": "No authorization key has been configured."
    }, 
    "dataservicehandler.delete.status.no_such_user": {
        "message": "No such user defined in remote service."
    }, 
    "dataservicehandler.delete.status.success": {
        "message": "All account and PhantomBot data has been removed from remote server."
    }, 
    "dataservicehandler.update.status.authentication_failed": {
        "message": "Authentication failed, check API key."
    }, 
    "dataservicehandler.update.status.launch_fail": {
        "message": "Failed to launch the remote API."
    }, 
    "dataservicehandler.update.status.no_auth_key": {
        "message": "No authorization key has been configured."
    }, 
    "dataservicehandler.update.status.no_such_user": {
        "message": "No such user defined in remote service."
    }, 
    "dataservicehandler.update.status.rate_limit": {
        "message": "Failed to update due to rate limits."
    }, 
    "dataservicehandler.update.status.success": {
        "message": "Data has been sent and loaded."
    }
}